src/styles/class-overrides.scss

```diff
@@ -0,0 +1,128 @@
+.usa-prose,
```

Comment:
Is there a specific way to organize this? Like in alphabetical order?

Reply:
Good point - I'll look into that